text_generation/causal_lm/cpp/continuous_batching/library/include/continuous_batching_pipeline.hpp

```diff
@@ -11,6 +11,13 @@
 #include "generation_config.hpp"
 #include "generation_handle.hpp"
 
+struct PipelineMetrics {
+    uint64_t m_step_counter = 0;
+    size_t m_total_requests = 0;
+    size_t m_scheduled_requests = 0;
+    float m_cache_usage = 0.0;
```

Comment:
I don't see it's set somewhere, but we have such value in BlockAllocator

Reply:
Yes, I'll add that in next commits